app/src/stores/server.ts

```diff
@@ -28,7 +28,7 @@ export type Info = {
 		theme_light_overrides: Record<string, unknown> | null;
 		theme_dark_overrides: Record<string, unknown> | null;
 		public_foreground: string | null;
-		public_background: string | null;
+		public_background: { id: string; type: string } | null;
```

Comment:
As of right now we gotta update changes to server infos both here and the sdk, else theyre out of sync, see:

https://github.com/directus/directus/blob/5153f53574d86ad6e3768280a1c86459c20651dc/sdk/src/schema/settings.ts#L13-L15